rc/lsp.kak

```diff
@@ -1415,6 +1423,49 @@
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+define-command -hidden lsp-indent-guides-request -docstring "lsp-indent-guides-request: request updating lsp_indent_guides for the buffer" %{
+  declare-option -hidden int lsp_indent_guides_timestamp -1
+  nop %sh{
+    if [ $kak_opt_lsp_indent_guides_timestamp -eq $kak_timestamp ]
+    then
+      exit
+    fi
+
+    indent_guides_characters="[ $(
+      DELIM=""
+      for CHARACTER in $(echo $kak_quoted_opt_lsp_indent_guides_characters)
```

Comment:
> echo $kak_quoted_opt_lsp_indent_guides_characters

this kind of expansion has some pitfalls, for example it won't work on spaces.
It doesn't matter here usually but it's user input so better use this pattern:

	eval set -- "$kak_quoted_opt_lsp_indent_guides_characters"
	for CHARACTER # in "$@" (but that's implied)
	do
		...
	done

while at it, we can maybe simplify it (not sure if the trailing comma is okay, didn't test it)

	eval set -- "$kak_quoted_opt_lsp_indent_guides_characters"
	printf %s, "$@"

Reply:
Done, thanks